.netlify/state.json

```diff
@@ -0,0 +1,3 @@
+{
+	"siteId": "7477dfa0-78a2-4749-92a1-9c1eef30ad89"
```

Comment:
I started using `netlify dev` and linked this repo to the netlify platform and that caused this file to be added. Do we keep it or ignore it @jbolda >